README.md

```diff
@@ -310,6 +310,24 @@ The API Supports a number of options to configure the read
         </td>
         <td>Write</td>
      </tr>
+   <tr valign="top">
```

Comment:
Shouldn't we be documenting "loadSchemaUpdateOption" here since that is how we are exposing this feature to the users?

Reply:
We are not exposing `loadSchemaUpdateOption`, it's an internal representation. We are exposing the `allowFieldAddition` and `allowFieldRelaxation` options